VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.html

```diff
@@ -1,11 +1,9 @@
-<div>
-  <img
+<span
     appTooltip
     [text]="tooltip"
     (tooltipShown)="resetText()"
-    class="icons"
-    src="/assets/images/share.png"
     [attr.alt]="'copy-id.alt' | translate"
-    (click)="copyToClipboard(conference); $event.stopPropagation()"
-  />
-</div>
+    (click)="copyToClipboard(conference); $event.stopPropagation()">
+  <em class="fad fa-address-card" aria-hidden="true"></em>
```

Comment:
I notice some of the icons use em and some use i. We should probably stick to one and I think it should be i as this is pretty common to use for icons and has no semantic meaning. em has semantic meaning.

https://fontawesome.com/v5.15/how-to-use/on-the-web/referencing-icons/basic-use

Reply:
I only changed this because sonarcloud was failing as I used <i> here

Reply:
so like i said, sonarcloud simply doesnt allow this change here with the `<i>` element

Reply:
Did you have the aria-hidden attribute when they were <i>s? According to the documentation that should make it compliant.

https://sonarcloud.io/organizations/nicolas-harraudeau-sonarsource-g/rules?open=Web%3ABoldAndItalicTagsCheck&q=bold

Reply:
ok will try it out again

Reply:
looks like I was missing that attribute in the one of the component, works now